### PR TITLE
glimmerizes SessionsGrid components

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

File: addon/components/sessions-grid-last-updated.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import moment from 'moment';
 
-export default Component.extend({
-  classNames: ['sessions-grid-last-updated'],
-  session: null,
-  updatedAt: null,
-  didReceiveAttrs(){
-    this._super(...arguments);
-    this.set('updatedAt', moment(this.get('session.updatedAt')).format("L LT"));
+export default class SessionsGridOfferingTable extends Component {
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply addon/components/sessions-grid-last-updated.js]: doh! fixed.

File: addon/components/sessions-grid-loading.hbs [hunk 2]
```diff
@@ -1,31 +1,33 @@
 {{! template-lint-disable no-unused-block-params }}
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply addon/components/sessions-grid-loading.hbs]: corrected.

File: package.json [hunk 3]
```diff
@@ -59,11 +59,11 @@
     "ember-fetch": "^6.7.2",
     "ember-file-upload": "^2.5.0",
     "ember-intl": "^4.0.0",
-    "ember-lifeline": "^5.0.0",
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply package.json]: ah, that was a mistake, i must have yanked it with `ember-lifeline-decorators` which didn't work out. good thing you caught this. i'm adding this back in.

File: addon/components/sessions-grid.hbs [hunk 4]
```diff
@@ -1,55 +1,81 @@
...
+                  {{truncate obj.postrequisite.title 18 true}}
+                </LinkTo>
+              {{else}}
+                <strong>
+                  {{t "general.ilm"}}:
+                  {{t "general.dueBy"}}
+                </strong>
+                {{moment-format
+                  (await obj.session.firstOfferingDate) "MM/DD/YYYY"
+                  allow-empty=true
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply addon/components/sessions-grid.hbs]: i know right?!

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jrjohnson) [reply package.json]: 👍

**Pull request merged**